gradle.properties

```diff
@@ -41,4 +41,4 @@ curseProjectId=240630
 curseHomepageLink=https://www.curseforge.com/minecraft/mc-mods/just-enough-resources-jer
 
 # Version
-specificationVersion=1.2.2
\ No newline at end of file
+specificationVersion=1.2.2.201
```

Comment:
Please remove .201, this part will be added automatically by Travis when compiling and publishing the mod

Reply:
Okay thanks for clarifying that, will fix the same things in the other PRs aswell